src/handle_action.cpp

```diff
@@ -177,21 +177,21 @@ input_context game::get_player_input( std::string &action )
         ctxt.set_iso( true );
         // The list of allowed actions in death-cam mode in game::handle_action
         // *INDENT-OFF*
-        for( const action_id id : {
+        for (const action_id id : {
```

Comment:
Looks like you need to run astyle on src/handle_action.cpp. (Yes, it's annoying, but it ultimately helps prevent spacing stuff like this from showing up in diffs.)

Reply:
It's weird, i'm running it but nothing is changing locally,

Reply:
Perhaps it isn't using the .astylerc in the CDDA main directory?

Reply:
I just checked and it's all equal, weird

Reply:
i tried styling it myself and there don't seem to be any changes locally. there's something afoot.

Reply:
Could it be some configuration or something i did in my part?

Reply:
Well, nothing i did changed, so i think if no one has anything else i think i will mark as resolved because i don't know what else to do